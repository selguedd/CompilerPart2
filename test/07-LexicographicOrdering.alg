VARIABLES NOT in lexicographic order
