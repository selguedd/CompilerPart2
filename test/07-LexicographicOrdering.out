token: VARIABLES	lexical unit: VARNAME
token: NOT	lexical unit: VARNAME
token: in	lexical unit: VARNAME
token: lexicographic	lexical unit: VARNAME
token: order	lexical unit: VARNAME
Identifiers
NOT	1
VARIABLES	1
in	1
lexicographic	1
order	1
